WebDriverTests/imported/selenium/py/conftest.py

```diff
@@ -49,6 +49,10 @@ def pytest_addoption(parser):
                      help='driver to run tests against ({})'.format(', '.join(drivers)))
     parser.addoption('--browser-binary', action='store', dest='binary',
                      help='location of the browser binary')
+    parser.addoption('--browser-target-ip', action='store', dest='browser_ip',
+                     help='ip of exiting browser instance running in automation mode')
+    parser.addoption('--browser-target-port', action='store', dest='browser_port',
+                     help='port of exiting browser instance running in automation mode')
```

Comment:
We should not change the imported selenium, except when updating from upstream. Are these changes upstream already? or proposed?

Reply:
These options are mandatory in order to tests to work. Without these we cannot connect to running remote instance and if these are not present and we still pass them from test suite executor then test fails when trying to parse non supported options.
Some of the changes in imported selenium are obsolete if we take upstream but for example these options need to be added. I'm going to propose all of the needed changes to upstream. We could wait for upstream PRs to land and then update imported scripts but not sure how long will it take. Also without these parameters we cannot land wk test suite script changes in this PR

Reply:
ok let's try pushing changes in imported folder to selenium upstream first

Reply:
Regarding the `Tools/` and `WebDriverTests/` changes, what if we remove them, leaving the patch only adding the options to the WebDriver service? As this feature is not exercised by the bots, we can still test it manually, unblocking the patch while the Selenium part is sorted out.

Once we update `WebDriverTests/imported/selenium`, we can add the `Tools/` changes adding the new options back in a separate PR.

Reply:
Sure, it's ok to land the reest of the patch

Reply:
Ok, patch is now update and Tools and WebDriverTests changes are now removed. Will push them later when we get support to selenium